src/util/constants.ts

```diff
@@ -37,11 +37,11 @@ export const MAX_ZOOM = 12;
 export const MAP_COLORS = ['#0078C0', '#FF9800', '#1976D2', '#D32F2F', '#E040FB', '#FF4081', '#512DA8', '#536DFE', '#F57C00', '#E64A19', '#8BC34A', '#689F38', '#FFC107', '#CDDC39'];
 
 /**
- * Url of the paikkatietovipunen
+ * Url of the ptp.hel.fi API
  * @readonly
  * @const {string}
  */
-export const PAIKKATIETOVIPUNEN_URL = 'http://paikkatietovipunen.hel.fi:10058';
+export const PTP_URL = 'https://ptp.hel.fi';
```

Comment:
Here it would be really nice if we could figure out what PTP stands for and write it out in the variable to describe what this is

Reply:
Renamed PTP_URL to PAIKKATIETOPALVELU_URL